.github/workflows/enhanced_gradients-publish.yml

```
@@ -0,0 +1,39 @@
+name: enhanced_gradients publish
+
+on:
+  push:
+    tags: ["enhanced_gradients-v*"]
+
+jobs:
+  publish:
+    name: Publish to pub.dev
+
+    runs-on: ubuntu-latest
+
+    permissions:
+      id-token: write
+      contents: write
+
+    defaults:
+      run:
+        working-directory: packages/enhanced_gradients
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Dart
+        uses: dart-lang/setup-dart@v1
+        with:
+          sdk: 3.0.0
+
+      - name: Set up Flutter
+        uses: subosito/flutter-action@v2
+        with:
+          flutter-version: 3.10.0
```

```suggestion
flutter-version: 3.10.x
```

a bit sad that [Dart action doesn't support this](https://github.com/dart-lang/setup-dart/issues/56)